File: javascript/1079.js
var inp = require('fs').readFileSync('/dev/stdin', 'utf8');
var values=inp.split('\n')
var q=parseInt(values.shift())
for(let i=1;i<=q;i++){
    var v=values.shift().split(" ")
    var [a,b,c]=v;
    a*=2
    b*=3
    c*=5
    media=((a+b+c)/10).toFixed(1)
    console.log(media)
}

File: C/1079.c
#include <stdio.h>
 
int main() {
    int inp;
    float nota,media,a,b,c;
    scanf("%i",&inp);
    for(int i=1;i<=inp;i++){
        scanf("%f %f %f",&a,&b,&c);
        media=(a*2+b*3+c*5)/10.0;
        printf("%.1f\n",media);
    }
    return 0;
}